syntax/rspec.vim

```diff
@@ -4,7 +4,7 @@
 "
 "
 
-runtime! syntax/ruby.vim
```

Comment:
So do we not need this assuming that it's going to be the current syntax? And then that will be sourced automatically? And this case works for all 3?